BP/common/culture/cultures/00_iberian.txt

```diff
@@ -0,0 +1,1096 @@
+			"dynn_FLA_inez"
+			"dynn_Froilaz"
+			"dynn_GarcI_a"
+			"dynn_GodestE_iz"
+			"dynn_GO_mez"
+			"dynn_GonzA_lez"
+			"dynn_GundemA_riz"
+			"dynn_GuillE_n"
+			"dynn_GutiE_rrez"
+			"dynn_LO_pez"
+			"dynn_MartI_n"
+			"dynn_MelE_ndez"
+			"dynn_MuN_iz"
+			"dynn_NU_N_ez"
+			"dynn_OrdO_N_ez"
+			"dynn_Osorio"
+			"dynn_OvE_quiz"
+			"dynn_PelA_ez"
+			"dynn_PE_rez"
+			"dynn_PiN_ole"
+			"dynn_Ponce"
+			"dynn_PurcE_lliz"
+			"dynn_QuiN_ones"
+			"dynn_RodrI_guez"
+			"dynn_SA_nchez"
+			"dynn_SebastiA_nez"
+			"dynn_SolI_s"
+			"dynn_SuA_rez"
+			"dynn_YA_N_ez"
+			"dynn_Vela"
+			"dynn_VE_laz"
+			"dynn_VelA_zquez"
+			{ "dynnp_de" "dynn_Aguilar" }
+			{ "dynnp_de" "dynn_Bembibre" }
+			{ "dynnp_de" "dynn_Cachero" }
+			{ "dynnp_de" "dynn_Canales" }
+			{ "dynnp_de" "dynn_Candamu" }
+			{ "dynnp_de" "dynn_CarriO_n" }
+			{ "dynnp_de" "dynn_Corias" }
+			{ "dynnp_de" "dynn_Entralgo" }
+			{ "dynnp_de" "dynn_Ferrera" }
+			{ "dynnp_de" "dynn_Granda" }
+			{ "dynnp_de" "dynn_Ledesma" }
+			{ "dynnp_de" "dynn_LeO_n" }
+			{ "dynnp_de" "dynn_Mancilla" }
+			{ "dynnp_de" "dynn_Mier" }
+			{ "dynnp_de" "dynn_Navia" }
+			{ "dynnp_del" "dynn_Ponte" }
+			{ "dynnp_de" "dynn_QuirO_s" }
+			{ "dynnp_de" "dynn_RiaN_o" }
+			{ "dynnp_de" "dynn_Robles" }
+			{ "dynnp_de" "dynn_SaldaN_a" }
+			{ "dynnp_de" "dynn_Santalla" }
+			{ "dynnp_de" "dynn_SantibA_N_ez" }
+			{ "dynnp_de" "dynn_Tabladiello" }
+			{ "dynnp_de" "dynn_Oviedo" }
+			{ "dynnp_de" "dynn_Valdivia" }
+			{ "dynnp_del" "dynn_Valle" }
+			{ "dynnp_de_la" "dynn_Vega" }
+			{ "dynnp_de" "dynn_Villagra" }
+			{ "dynnp_de" "dynn_Villalba" }
+			{ "dynnp_de" "dynn_Villanueva" }
+			{ "dynnp_de" "dynn_Villalobos" }
+		}
+
+		male_names = {
+			Alfonsu A_lvaru AndrE_s Arias Asur Aureliu Bermudu Bernaldu BosO_n Diegu
+			Domingu EstE_banu Facundu Favila Fernandu FlaI_n Fruela GarcI_a GO_mez Gonzalu
+			GuillE_n Gundemaru Gutierre Hermenexildu Llope LloriA_n Manuel MartI_n Melendu Muniu
+			NuN_u OrdoN_u Osoriu Ovecu Pedru Pelayu PiN_olu Ponce Ramiru RamO_n
+			Rodrigu Sanchu Savaricu SebastiA_n Silu Sueru Vela Velascu Xerardu Xil
+			Xuan
+		}
+		female_names = {
+			Adosinda Aldara Aldonza Aragonte Beatriz Belasquida Blanca Catalina Constanza Cristina
+			Dulce Egilona Elvira Enderquina Ermesenda EstefanI_a Eva Fronilde Gontroda Gotina
+			Goto Guiomar InEEs Iyana Leodegundia Lleonor Margarita MarI_a Marina Marquesa
+			Mayor MencI_a MI_lia Munia Muniadona Nicor NuN_a Olaya Oneca Patrona
+			Sabela Sancha Taresa Tegridia Toda Urraca Varela Ximena Xuana Xusta
+		}
+
+		dynasty_of_location_prefix = "dynnp_de"
+
+		patronym_suffix_male = "dynnpat_suf_ez"
+		patronym_suffix_female = "dynnpat_suf_ez"
+		always_use_patronym = yes
+
+		pat_grf_name_chance = 45
+		mat_grf_name_chance = 15
+		father_name_chance = 30
+
+		pat_grm_name_chance = 40
+		mat_grm_name_chance = 10
+		mother_name_chance = 40
+
+		ethnicities = {
+			10 = mediterranean
+		}
+	}
+
+	aragonese = { # Unused in our files
```

Comment:
The reason for why it is unused is because it new with CK3, I believe ?

Reply:
Yeah